package.json

```diff
@@ -22,9 +22,9 @@
     "clean": "rm -rf dist/ node_modules/",
     "lint": "eslint --max-warnings 0 .",
     "lint:format": "prettier --check .",
-    "start": "yarn build --watch",
+    "start": "pnpm run build --watch",
```

Comment:
Have you tested if this is working? Don't you need to use `pnpm run build -- --watch`

Reply:
Good catch! Thanks